kano_apps/AppManage.py

```diff
@@ -67,7 +67,7 @@ def uninstall_packages(app, sudo_pwd=None):
 
     pkgs = " ".join(app["packages"])
 
-    cmd = "rxvt -title 'Uninstalling {}' -e bash -c ".format(app["title"])
+    cmd = ("rxvt -title " + _("'Uninstalling {}'") + " -e bash -c ").format(app["title"])
```

Comment:
There is a problem here, in that os.system expects ordinary strings, so we need to do `os.system(cmd.encode('utf8'))` below, and similarly elsewhere in this file. I've made this change locally.